test/libsolidity/SemanticTest.cpp

```diff
@@ -175,11 +176,43 @@ map<string, Builtin> SemanticTest::makeBuiltins()
 				return toBigEndian(h256(ExecutionFramework::setAccount(accountNumber).asBytes(), h256::AlignRight));
 			}
 		},
+		{
+			"bytecode",
+			[this](FunctionCall const& _call) -> optional<bytes>
+			{
+				bool result = false;
+				optional<regex> regexPattern;
+				soltestAssert(m_deployedBytecode.has_value(), "No bytecode deployed.");
+				soltestAssert(_call.arguments.parameters.size() == 1, "Bytecode pattern string expected.");
+				string pattern = _call.arguments.parameters.at(0).rawString;
+				bytes bytecode = m_deployedBytecode.value();
+				if (pattern.length() >= 2 && boost::starts_with(pattern, "\"") && boost::ends_with(pattern, "\""))
+				{
+					pattern = pattern.substr(1, pattern.length() - 2);
+					try
+					{
+						regexPattern = regex(pattern);
+					}
+					catch (std::regex_error const&)
+					{
+					}
```

Comment:
Can you give an example that falls into this `catch` and still passes the `isValidHex(pattern)` check below? I hink that all the possible hex patterns are valid regexes.